test/test.ts

```diff
@@ -46,6 +45,9 @@ class ConfigStore {
   }
 }
 
+const RESUMABLE_INCOMPLETE_STATUS_CODE = 308;
```

Comment:
Just for consistency I noticed a few other places are checking status codes directly against a hardcoded number. It might make sense to either add them as constants or remove this and just compare directly to the code.

Reply:
Never mind I just noticed this was in the tests file.